### PR TITLE
Update build badges

### PR DESCRIPTION
**Pull request by SimonBiggs**
@pchlap did you want to jump in and review a few of these for me?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -9,13 +9,16 @@
...
 .. |streamlit| image:: https://static.streamlit.io/badges/streamlit_badge_black_white.svg
     :target: https://share.streamlit.io/pymedphys/pymedphys/main/app.py
 
-.. |build| image:: https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2Fpymedphys%2Fpymedphys%2Fbadge&label=build&logo=none
-    :target: https://actions-badge.atrox.dev/pymedphys/pymedphys/goto
+.. |build| image:: https://img.shields.io/github/workflow/status/pymedphys/pymedphys/PullRequest
+    :target: https://github.com/pymedphys/pymedphys/actions
+
+.. |docs| image:: https://img.shields.io/netlify/c702e3b2-f436-46a3-b461-00a8a55bcdba
+    :target: https://app.netlify.com/sites/pymedphys/deploys
```
Comment (SimonBiggs) [inline README.rst:21]: Will flag if netlify docs are not building.

File: README.rst [hunk 2]
```diff
@@ -9,13 +9,16 @@
...
-|streamlit| |build| |pypi| |python| |license|
+|streamlit| |build| |docs| |pypi| |python| |license|
 
 .. |streamlit| image:: https://static.streamlit.io/badges/streamlit_badge_black_white.svg
     :target: https://share.streamlit.io/pymedphys/pymedphys/main/app.py
 
-.. |build| image:: https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2Fpymedphys%2Fpymedphys%2Fbadge&label=build&logo=none
-    :target: https://actions-badge.atrox.dev/pymedphys/pymedphys/goto
+.. |build| image:: https://img.shields.io/github/workflow/status/pymedphys/pymedphys/PullRequest
+    :target: https://github.com/pymedphys/pymedphys/actions
```
Comment (SimonBiggs) [inline README.rst:18]: Old badge appeared to be stuck pointing to "master"

**Review by pchlap — APPROVED** [review]
Not much to review here, but I checked the links and all looks good to me :)

**Comment by SimonBiggs** [comment]
Thank @pchlap :)

**Pull request merged**